Assets/MixedRealityToolkit/InputModule/Scripts/Editor/XboxControllerInputSourceEditor.cs

```diff
@@ -1,14 +1,15 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
+using MixedRealityToolkit.InputModule.InputSources;
 using System;
 using UnityEditor;
 using UnityEngine;
 
-namespace HoloToolkit.Unity.InputModule
+namespace MixedRealityToolkit.InputModule.Editor
 {
     [CustomEditor(typeof(XboxControllerInputSource))]
-    public class XboxControllerInputSourceEditor : Editor
+    public class XboxControllerInputSourceEditor : UnityEditor.Editor
```

Comment:
I'm not sure how I feel about this namespace because it hides the UnityEditor one.

Reply:
Like Cursor.  It doesn't block anything in the UnityEngine.Editor namespace as we are only extending it.
But having this highlights those scripts which are truly dependant on Unity "stuff"